lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -51,6 +55,48 @@ defmodule Trento.Clusters do
     |> Repo.preload([:tags, :hosts_executions, :checks_results])
   end
 
+  @spec get_hosts_connection_settings(String.t()) :: [
+          %{
+            host_id: String.t(),
+            hostname: String.t(),
+            user: String.t()
+          }
+        ]
+  def get_hosts_connection_settings(cluster_id) do
```

Comment:
We already have a `get_connection_settings` function in the hosts usecase. It takes the host id as param, that's true.
Now I see this, I wonder if we should maybe remove that one, and only use this, as they do more or less the same, and we even have the `cluster_id` in the place where the old one is used:
https://github.com/trento-project/web/blob/main/lib/trento/application/event_handlers/checks_event_handler.ex#L25
https://github.com/trento-project/web/blob/main/lib/trento/application/usecases/hosts/hosts.ex#L42

Reply:
Yes I saw that, wanted to fix, but it's a kinda out of scope for what I am doing.
Feel free to fix that, otherwise I'll get back to that next round.